### DISCUSSION TITLE
An error occurred while submitting the form.

### DISCUSSION THREAD

**Discussion by alexklann** [pocketbase/pocketbase#2269] [Q&A] [answered]
I am currently trying to get the email verification setup, and I am having this issue where inputting my own email into the form and then submitting - doesn't matter whether I am using plain authentification or login authentification - I am getting error 400 in the console and pocketbase is telling me that 'An error occurred while submitting the form.'

Anyone ever had this before? Using v0.14.3

**Comment by ganigeorgiev** [comment, marked as answer]
There are not enough details to help you debug the issue.

You can start the executable with `--debug` and check for any additional error messages and/or check in the Admin UI > Logs the failed request `meta` data.

If the above doesn't help, please provide at least a code sample how you are sending the request.

**Comment by alexklann** [comment]
Okay so I switched from the docker version of PocketBase to the single executable version and now it is working for some odd reason.
